src/standalone/Vegetation/Canopy.jl

```diff
@@ -521,6 +539,9 @@ function CanopyModel{FT}(;
     photosynthesis::AbstractPhotosynthesisModel{FT},
     conductance::AbstractStomatalConductanceModel{FT},
     hydraulics::AbstractPlantHydraulicsModel{FT},
+    soil_moisture_stress::AbstractSoilMoistureStressModel{FT} = TuzetMoistureStressModel{
```

Comment:
e.g. here the default does not match the default in the LandModel constructor you changed.

Reply:
I also think we should change the default to be the Piecewise approach, but that can be a next PR so this PR is not behavior changing